### PR TITLE
Improved chart functionality for US7

### PR DESCRIPTION
**Pull request by Paciflow**

### CODE REVIEW

**Comment by Paciflow** [comment]
Fix: Installed chart dependencies in the correct directory and resolved build issues

**Review by LouisChoiniere — CHANGES_REQUESTED** [review]
This is a very good start, but the backend is already created please use the backend endpoints

**Comment by Paciflow** [comment]
Changes Made: 
Axis Labels: for the x-axis (Team Members) and y-axis (Scores).
Font Contrast: larger font size.
Scaling: so the chart doesn’t look cramped.

File: CodebustersPAS-ClientApp/src/pages/Teacher/InstructorDashboard.tsx [hunk 1]
```diff
@@ -1,24 +1,67 @@
...
-      <div className="links-container">
-        <a href="/summary" className="dashboard-link">Summary of results</a>
-        <a href="/detailed-summary" className="dashboard-link">Detailed Summary of results</a>
-        <a href="/charts" className="dashboard-link">Charts</a>
-        <a href="/Teacher/TeamOverview" className="dashboard-link">Return to main menu</a>
-      </div>
+    <div className="App">
+      <Navbar showLogout={showLogout} />
+      <div className="content"></div>
+      <Routes>
```
Comment (LouisChoiniere) [inline CodebustersPAS-ClientApp/src/pages/Teacher/InstructorDashboard.tsx]: Why did you add the router from app.tsx to this component?

**Review by LouisChoiniere — CHANGES_REQUESTED** [review]

File: CodebustersPAS-ClientApp/src/pages/Teacher/InstructorDashboard.tsx (hunk 1 above)
Comment (Paciflow) [reply CodebustersPAS-ClientApp/src/pages/Teacher/InstructorDashboard.tsx]: > Why did you add the router from app.tsx to this component?

Yeah, it shouldn't have been there. I took that part out.

**Review by LouisChoiniere — APPROVED** [review]
still not linked to backend but looks good

**Review by OmarSelim01 — APPROVED** [review]
